server/buttonAlerts.js

```diff
@@ -13,12 +13,12 @@ function setup(braveAlerterObj) {
   braveAlerter = braveAlerterObj
 }
 
-async function handleValidRequest(button, numButtonPresses) {
+async function handleValidRequest(button, numberOfAlerts) {
   // Log the request
   helpers.log(
     `id: ${button.id.toString()} SerialNumber: ${
       button.buttonSerialNumber
-    } Unit: ${button.displayName.toString()} Presses: ${numButtonPresses.toString()} Is Sending Alerts?: ${
+    } Unit: ${button.displayName.toString()} Presses: ${numberOfAlerts.toString()} Is Sending Alerts?: ${
```

Comment:
Similarly, `numButtonPresses` is always 1 here now that we're using RAK buttons. So you can probably take this out of the log entirely.